pkg/controller/nodesetup/conditions.go

```diff
@@ -3,6 +3,8 @@
 package nodesetup
 
 const (
+	// TODO(rzetelskik): change format to NodeSetup in next API version
```

Comment:
add issue ref where further discussion can be found and motivation behind the change can be explained